src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -255,4 +224,29 @@ public ExerciseResource(ExerciseRepository exerciseRepository, ExerciseService e
             .body(resource);
     }
 
+    /**
```

Comment:
I would move this code up, so that GET is placed before all the delete, reset and cleanup methods

Reply:
Done